cmd/exec.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var execCmd = &cobra.Command{
+	Use:   "exec <containerID> <command>",
+	Short: "exec a command in a running container",
+	Args:  cobra.ExactArgs(2),
+	Run:   execCommand,
+}
+
+func execCommand(_ *cobra.Command, args []string) {
```

Comment:
This just spawns a process in the container directory, but it doesn't actually exec "inside" the container